sdk/communication/communication-common/review/communication-common.api.md

```diff
@@ -70,6 +73,9 @@ export interface EndpointCredential {
 // @public
 export const getIdentifierKind: (identifier: CommunicationIdentifier) => CommunicationIdentifierKind;
 
+// @public
+export const getRawId: (identifier: CommunicationIdentifier) => string;
```

Comment:
Other naming candidates:
```ts
getRawIdForIdentifier // longer but more explicit, doesn't match getIdentifierKind which isn't named. "For" because it is used to describe the identifier.
getIdentifierKindForIdentifier
getRawIdOfIdentifier // "of" is a little more clear that every identifier always has a corresponding rawId
getRawIdFromIdentifier // "from" makes it sound that the rawId is derived. Which is true but conceptually the rawId always exists no matter if created here or not
createRawIdFromIdentifier // create sounds heavy for a string, and conceptually the underlying raw id exists independent from creating it with this helper
```

Reply:
More options:
```ts
toRawId
fromRawId
```

Reply:
Prefer the names that start with `get*` because it matches `getIdentifierKind`.

Weakly avoid `getRawId` because it's a bit too generic for communication-common. Taking up that name might add confusion if the common library ever gains another concept that could also have a raw id. `getIdentifierKind` doesn't have this issue as much because the noun `identifier` already has fixed semantics in this library.

So I'd choose between `getRawIdForIdentifier` and `getRawIdOfIdentifier`. (Very) weak preference for the first because the latter makes me think we're reading a field in an object.

Reply:
Some more suggestions:

1.  `serializeCommunicationIdentifier`  / `deserializeCommunicationIdentifier`
2.  `stringifyCommunicationIdentifier` / `parseCommunicationIdentifier` (homage to JSON.stringify / parse)

Reply:
how about `getIdentifierRawId`? it's more specific and in line with `getIdentifierKind`

Reply:
We already have internal `serializeCommunicationIdentifier`/`deserializeCommunicationIdentifier` which we use to translate between the SDK model and the REST API model.
The `stringify` / `parse` is interesting but in the other languages we will mostly expose a `RawId` getter property on the CommunicationIdentifier base class (which we don't have in JS). Using the term rawId then seems more consistent.

I like `getIdentifierRawId` because it will nicely group with `getIdentifierKind` in intellisense, thank you for that suggestion.